clang/unittests/Format/FormatTest.cpp

```diff
@@ -20875,13 +20875,15 @@ TEST_F(FormatTest, CatchAlignArrayOfStructuresRightAlignment) {
                "};",
                Style);
   // TODO: Fix the indentations below when this option is fully functional.
+#if 0
```

Comment:
Why did you remove this?

Reply:
Because it's wrong as the TODO comment says.

This patch doesn't fix it and would format it as:
```
int a[][] = {
    {
     {0, 2}, //
        {1, 2}  //
    }
};
```
Nevertheless, I just changed it to what it ought to look like.